src/Commands/ArtCommand.php

```diff
@@ -23,17 +51,37 @@ class ArtCommand extends TerminusCommand
      *
      * @param string $name Artwork name
      *
-     * @usage Displays the list of available artwork.
      * @usage <artwork> Displays the <artwork> artwork.
      */
-    public function art($name = 'random')
+    public function art($name = 'random', array $options = ['list' => false,])
```

Comment:
what's this $options do?  it doesn't seem to be used in the function body.  Is this a vestige from an earlier approach?

Reply:
Whoops, yes. I had a lingering change I forgot to commit - updated that.